core/src/main/java/io/varhttp/VarConfig.java

```diff
@@ -14,4 +16,13 @@ public VarConfig setPort(int port) {
 		this.port = port;
 		return this;
 	}
+
+	public Optional<Boolean> getSecureContext() {
```

Comment:
Add javadoc, and think about better name

Reply:
forceSecureServer ?

Reply:
Good, should be just boolean and not optional of bool?

Reply:
the idea was that this will override all the configs? and if was not set then will check other stuff... but I can change logic. 

Also the naming that was thinking: 

`isRequestSecure` and `forceRequestSecure`

Reply:
@nut-neek now I understood your point! Change it to type `Boolean`. I thought that Boolean couldn't be null but it can!